bookrack-backend/src/test/java/org/example/bookrackbackend/BookControllerTest.java

```diff
@@ -95,14 +95,6 @@ void getBookById_shouldThrowException_whenIdNotFound() throws Exception {
 
     }
 
-    @Test
-    void getBookById_shouldThrowException_whenInvalidIdProvided() throws Exception {
-        mockMvc.perform(MockMvcRequestBuilders.get("/api/books/#")).andExpect(status().isNotFound())
-                .andExpect(content().json("""
-                {"message": "Please check entered URL."}
-                """));
-
-    }
```

Comment:
Why have you removed this test?

A check for invalid ID seems sensible, but I would expect another response message.